controllers/grid/pubIds/PubIdExportSubmissionsListGridHandler.inc.php

```diff
@@ -0,0 +1,299 @@
+<?php
+
+/**
+ * @file controllers/grid/pubIds/PubIdExportSubmissionsListGridHandler.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2000-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class PubIdExportSubmissionsListGridHandler
+ * @ingroup controllers_grid_pubIds
+ *
+ * @brief Handle exportable submissions with pub ids list grid requests.
+ */
+
+import('lib.pkp.classes.controllers.grid.GridHandler');
+
+import('lib.pkp.controllers.grid.submissions.exportableSubmissions.ExportableSubmissionsGridRow');
+import('controllers.grid.pubIds.PubIdExportSubmissionsListGridCellProvider');
+
+class PubIdExportSubmissionsListGridHandler extends GridHandler {
+	/** @var boolean true if the current user has a managerial role */
+	var $_isManager;
+
+	/** @var ImportExportPlugin */
+	var $_plugin;
+
+	/**
+	 * Constructor
+	 */
+	function PubIdExportSubmissionsListGridHandler() {
+		parent::GridHandler();
+		$this->addRoleAssignment(
+			array(ROLE_ID_MANAGER),
+			array('fetchGrid', 'fetchRow')
+		);
+	}
+
+	//
+	// Implement template methods from PKPHandler
+	//
+	/**
+	 * @copydoc PKPHandler::authorize()
+	 */
+	function authorize($request, &$args, $roleAssignments) {
+		import('lib.pkp.classes.security.authorization.ContextAccessPolicy');
+		$this->addPolicy(new ContextAccessPolicy($request, $roleAssignments));
```

Comment:
Isn't that an overly vague policy (e.g. permitting anyone with context access to view this)?